README.md

```diff
@@ -29,3 +29,18 @@ See some of the many HDFS tutorials out there for more details, but here's a qui
 1. `hadoop fs -ls hdfs://<ActiveNameNode>:50071/` should show nothing for starters
 2. `hadoop fs -put /path/to/src_file hdfs://<ActiveNameNode>:50071/`
 3. `hadoop fs -ls hdfs://<ActiveNameNode>:50071/` should now list src_file
+
+Resource Reservation Instructions (Optional)
+--------------------------
+
+1. On master, add the role for HDFS, by running `echo hdfs > /etc/mesos-master/role`
+2. Then restart the master by running `sudo service mesos-master restart`
+3. On each slave where you want to reserve resources, add specific resource reservations for the HDFS role. Here is one example:
+<br>`cpus(*):2;cpus(hdfs):2;mem(*):4096;mem(hdfs):4096 > /etc/mesos-slave/resources`</br>
```

Comment:
Maybe give an example where the values aren't the same, so it isn't assumed that is a requirement